src/Umbraco.Core/CodeAnnotations/UmbracoSuppressVolatileAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Umbraco.Core.CodeAnnotations
+{
+    /// <summary>
+    /// Assembly level attribute used to suppress <see cref="UmbracoVolatileAttribute"/> errors to warnings.
```

Comment:
I think we need more info in this file.. More info about it is used by Umbraco.Core package etc. bla.bla.bla :) Otherwise, some would think it could be removed 🙈

Reply:
That's very true, I added a comment to the top of the file explaining what its purpose is :)